gerrymandering/gerrymandering.ipynb

```diff
@@ -32,8 +32,10 @@
     "* [Numba](http://numba.pydata.org): Compiles low-level numerical code written in Python into very fast machine code\n",
     "* [Dask](http://dask.pydata.org): Distributes these numba-based workloads across multiple processing cores in your machine\n",
     "* [Datashader](http://datashader.org): Using Numba and Dask, aggregates big datasets into a fixed-sized array suitable for display in the browser\n",
-    "* [GeoViews](http://geoviews.org/) (using [Cartopy](http://scitools.org.uk/cartopy)): Project longitude, latitude shapes into Web Mercator and create visible objects\n",
+    "* [GeoViews](http://geoviews.org/): To create points for the color key\n",
```

Comment:
I believe GeoViews is for properly projecting and visualizing points onto a geographic map.